platforms/native/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/gcc/AbstractGccCompatibleToolChain.java

```diff
@@ -96,6 +96,7 @@ public AbstractGccCompatibleToolChain(String name, BuildOperationExecutor buildO
         target(new Intel32Architecture());
         target(new Intel64Architecture());
         target(new OsxArm64Architecture());
+        target(new LinuxArm64Architecture());
```

Comment:
Instead of adding another class, could we rename `OsxArm64Architecture` to `Arm64Architecture` and in `supportsPlatform`, check for Linux or macOS?

Reply:
done